x-pack/plugins/security_solution/public/timelines/components/field_renderers/more_container/index.test.tsx

```diff
@@ -0,0 +1,130 @@
+  describe('MoreContainer', () => {
+    const idPrefix = 'prefix-1';
+    const rowItems = ['item1', 'item2', 'item3', 'item4', 'item5', 'item6', 'item7'];
+
+    test('it should only render the items after overflowIndexStart', () => {
+      render(
+        <TestProviders>
+          <MoreContainer
+            idPrefix={idPrefix}
+            overflowIndexStart={5}
+            values={rowItems}
+            fieldName="mock.attr"
+          />
+        </TestProviders>
+      );
+
+      expect(screen.getByTestId('more-container').textContent).toEqual('item6item7');
+    });
+
+    test('it should render all the items when overflowIndexStart is zero', () => {
+      render(
+        <TestProviders>
+          <MoreContainer
+            idPrefix={idPrefix}
+            overflowIndexStart={0}
+            values={rowItems}
+            fieldName="mock.attr"
+          />
+        </TestProviders>
+      );
+
+      expect(screen.getByTestId('more-container').textContent).toEqual(
+        'item1item2item3item4item5item6item7'
+      );
+    });
+
+    test('it should have the eui-yScroll to enable scrolling when necessary', () => {
+      render(
+        <TestProviders>
+          <MoreContainer
+            idPrefix={idPrefix}
+            overflowIndexStart={5}
+            values={rowItems}
+            fieldName="mock.attr"
+          />
+        </TestProviders>
+      );
+
+      expect(screen.getByTestId('more-container')).toHaveClass('eui-yScroll');
+    });
+
+    test('it should use the moreMaxHeight prop as the value for the max-height style', () => {
+      render(
+        <TestProviders>
+          <MoreContainer
+            idPrefix={idPrefix}
+            overflowIndexStart={5}
+            moreMaxHeight={DEFAULT_MORE_MAX_HEIGHT}
+            values={rowItems}
+            fieldName="mock.attr"
+          />
+        </TestProviders>
+      );
+
+      expect(screen.getByTestId('more-container')).toHaveStyle(
+        `max-height: ${DEFAULT_MORE_MAX_HEIGHT}`
+      );
+    });
+
+    test('it should render with correct attrName prop', () => {
+      render(
+        <TestProviders>
+          <MoreContainer
+            idPrefix={idPrefix}
+            overflowIndexStart={5}
+            values={rowItems}
+            fieldName="mock.attr"
+          />
+        </TestProviders>
+      );
+
+      screen
+        .getAllByTestId('cellActions-renderContent-mock.attr')
+        .forEach((element) => expect(element).toBeInTheDocument());
+    });
+
+    test('it should only invoke the optional render function when provided', () => {
+      const renderFn = jest.fn();
+
+      render(
+        <TestProviders>
+          <MoreContainer
+            idPrefix={idPrefix}
+            overflowIndexStart={5}
+            render={renderFn}
+            values={rowItems}
+            fieldName="mock.attr"
+          />
+        </TestProviders>
+      );
+
+      expect(renderFn).toHaveBeenCalledTimes(2);
```

Comment:
Should it render twice or once?

Reply:
In this test the `renderFn` will be called twice because the `MoreContainer` component calls the render function for each item in the values array starting from `overflowIndexStart`. With `overflowIndexStart={5}` and an array length of 7, the `renderFn` is called twice (for 'value6' and 'value7').